presto-spi/src/main/java/com/facebook/presto/spi/PageSinkProperty.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi;
+
+public class PageSinkProperty
+{
+    public static final PageSinkProperty DEFAULT_PAGE_SINK_PROPERTY = new PageSinkProperty.Builder().build();
+
+    private final boolean lifespanCommitRequired;
+
+    private PageSinkProperty(boolean lifespanCommitRequired)
```

Comment:
@rschlussel I think we should take the same approach in the https://github.com/prestodb/presto/pull/12780 and introduce the SplitSourceProperties that will contain both, `splitSchedulingStrategy` and the `schedulerUsesAddresses`. With this approach the next time we want to pass something, we don't have to change the SPI.